modules/3d-tiles/src/lib/filesystems/tiles-3d-archive-file-system.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  FileProvider,
+  ZipFileSystem,
+  cdSignature as cdHeaderSignature,
+  searchFromTheEnd,
+  parseZipCDFileHeader,
+  HashElement,
+  parseHashFile,
+  parseZipLocalFileHeader
+} from '@loaders.gl/zip';
+import {TZ3Archive} from '../../tz3/tz3-archive';
```

Comment:
Nit: I personally think we should just drop the tz3 name completely. Given that it is a scrambled version of `3tz` it is just too obscure.
```suggestion
import {Tiles3DArchive} from '../../tiles-3d-archive/tiles-3d-archive';
```

Reply:
Will be done in another PR